### PR TITLE
Create lps_in_O(n).cpp

### PR DESCRIPTION
**Pull request by namanbansal013**

### CODE REVIEW

File: C++14/dp/lps_in_O(n).cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+//Returns Longest Palindrome Subsequence length. 
+//Time Complexity O(n).
+int LPSManchers(string input) {
```
Comment (likecs) [inline C++14/dp/lps_in_O(n).cpp:4]: In case this is taken from some submission, please mention the link else ignore this comment.

**Comment by likecs** [comment]
Thanks for the pull request. Changes look good, just minor comments.

Please address and then we are good to merge this one.

File: C++14/dp/lps_in_O(n).cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
+//Returns Longest Palindrome Subsequence length. 
+//Time Complexity O(n).
```
Comment (likecs) [inline C++14/dp/lps_in_O(n).cpp:2]: Please mention the space complexity as well.

File: C++14/dp/lps_in_O(n).cpp (hunk 1 above)
Comment (namanbansal013) [reply C++14/dp/lps_in_O(n).cpp:4]: No, this isn't taken from anywhere.

File: C++14/dp/lps_in_O(n).cpp (hunk 2 above)
Comment (namanbansal013) [reply C++14/dp/lps_in_O(n).cpp:2]: Done.

Comment (namanbansal013) [reply C++14/dp/lps_in_O(n).cpp:2]: Done with all the comments.

**Pull request merged**